pkg/services/vmoperator/vmopmachine.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Kubernetes Authors.
+Copyright 2021-2022 The Kubernetes Authors.
```

Comment:
no need to update this

Reply:
The file changed though, and dates are updated to reflect the most recent year, hence the range.